D3v4-Upgrade-Notes.md

```diff
@@ -33,3 +33,12 @@
 - Consider removing transitions for zooming and brushing
 - d3 stack to D3v4
 - Changing .tension call in lineChart
+
+
+### For end users
+
+- Changes to `xUnits` interface.
+- geoChart
+- Color scheme
+- tension in curves
+
```

Comment:
I am documenting all of these in Changelog.md, and also in the API documentation. We could do a D3-style CHANGES summary as well